### PR TITLE
5 get user from api

### LINKED ISSUE #5: Get User from our API
Write a function to get the user from our API - we may need to use superagent.

### PR DESCRIPTION
**Pull request by micah-peautagi**
Get users from api

closes #5

### CODE REVIEW

File: server/db/db.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const enviroment = process.env.NODE.ENV || 'development'
+const config = require('../knexfile.js')[enviroment]
+const connection = require('knex')(config)
+
+module.exports = {
```
Comment (bryceknz) [inline server/db/db.js]: Please extract these functions out into `users.js` and pull in the db config from `db/index.js`

File: client/api.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const userUrl = 'http://localhost:3000/api/v1'
```
Comment (bryceknz) [inline client/api.js]: The variable name `userUrl` implies that it will include the `/user` route. I would use the variable name `apiUrl` instead.

**Review by bryceknz — CHANGES_REQUESTED** [review]
Looks good! Just a couple of small changes. Please also add tests for this code.

File: server/db/db.js (hunk 1 above)
Comment (micah-peautagi) [reply server/db/db.js]: This is complete

File: client/api.js (hunk 2 above)
Comment (micah-peautagi) [reply client/api.js]: This is complete

**Comment by micah-peautagi** [comment]
Hey Bryce, test for API complete and ready for review. Cheers.

**Review by bryceknz — APPROVED** [review]

**Pull request merged**